dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/thread/ThreadPoolExecutorExt.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.core.internal.thread
+
+import java.util.concurrent.ThreadPoolExecutor
+import java.util.concurrent.TimeUnit
+
+internal const val MAX_SLEEP_DURATION_IN_MS = 25L
+
+internal fun ThreadPoolExecutor.waitToIdle(timeout: Long): Boolean {
```

Comment:
Let's rename the parameter to `timeoutMs` to make it explicit we expect a time in Ms